llama_stack/cli/stack/run.py

```diff
@@ -65,6 +65,13 @@ def _add_arguments(self):
             type=str,
             help="Path to TLS certificate file for HTTPS",
         )
+        self.parser.add_argument(
+            "--image-type",
+            type=str,
+            help="Image Type used during the build. This can be either conda or container or venv.",
```

Comment:
click already renders possible choices.

```
--image-type {conda,container,venv}
                        Image Type used during the build. This can be either conda or container or venv.
```